app.py

```diff
@@ -235,18 +252,12 @@ async def fetch_seed_json():
             result = response.read()
             result_json = json.loads(result.decode("utf-8"))
             SEED_CACHE = result_json
-
     except urllib.error.HTTPError as error:
         return jsonify({"error": f"Failed to retrieve the JSON. \
                         HTTP Status Code: {error.code}"}), 400
     except Exception as e:
         return jsonify({"error": str(e)}), 500
-
-
-@app.get("/health")
-async def health():
-    return NACHET_HEALTH_MESSAGE, 200
```

Comment:
As mentioned, no need to create an env variable for "ok"

Reply:
done